### ISSUE TITLE
Kinesis Event in Java Lambda contains only "kinesis" object data

### ISSUE THREAD

**Issue by raissaa** [localstack/localstack#2962]
# Type of request: This is a ...
[x] bug report
[ ] feature request

# Detailed description
Previously reported as https://github.com/localstack/localstack/issues/2938
When running a **Java 11 lambda** that processes a Kinesis event, the event only contains some of the "kinesis" object data.
I can see that the json that localstack generates to pass to the lambda contains most of the fields (as was addressed on https://github.com/localstack/localstack/pull/2946), but the KinesisEvent inside the lambda lacks data. 
I suspect the issue lies on the LambdaExecutor side as it only sets the specific 3 fields I get for a kinesis event: https://github.com/localstack/localstack-java-utils/blob/master/src/main/java/cloud/localstack/LambdaExecutor.java#L71

## Expected behavior
The records contain all information available for a kinesis event:

```
{
            "kinesis": {
                "kinesisSchemaVersion": "1.0",
                "partitionKey": "1",
                "sequenceNumber": "49590338271490256608559692538361571095921575989136588898",
                "data": "SGVsbG8sIHRoaXMgaXMgYSB0ZXN0Lg==",
                "approximateArrivalTimestamp": 1545084650.987
            },
            "eventSource": "aws:kinesis",
            "eventVersion": "1.0",
            "eventID": "shardId-000000000006:49590338271490256608559692538361571095921575989136588898",
            "eventName": "aws:kinesis:record",
            "invokeIdentityArn": "arn:aws:iam::123456789012:role/lambda-role",
            "awsRegion": "us-east-2",
            "eventSourceARN": "arn:aws:kinesis:us-east-2:123456789012:stream/lambda-stream"
}
```

## Actual behavior
The records only contain partitionKey, data and approximateArrivalTimestamp

```
{
            "kinesis": {
                "partitionKey": "1",
                "data": "SGVsbG8sIHRoaXMgaXMgYSB0ZXN0Lg==",
                "approximateArrivalTimestamp": 1545084650.987
            }
}
```

# Steps to reproduce
## Command used to start LocalStack
docker-compose up

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

┆Issue is synchronized with this [Jira Bug](https://localstack.atlassian.net/browse/LOC-259) by [Unito](https://www.unito.io/learn-more)

**Comment by ricard0javier** [comment]
Great! I was facing this problem yesterday and now I just confirmed that is an issue with localstack

**Comment by raissaa** [comment]
Hi :) Any news on this? Is it just updating the java lambda executor version at this point?

**Comment by raissaa** [comment]
Seems to be fixed - tested with 0.12.2
Thanks!

**Comment by philipholtseeland** [comment]
Hello, I am still experiencing a part of this issue, namely that the field _kinesisSchemaVersion_ does not seem to be included in the payload. Running LocalStack 1.3.1 with Testcontainers.

The payload looks like this:
```
{
    "Records":
    [
        {
            "eventID": "shardId-000000000000:49636781880378287419242807609792252425332679328602783746",
            "eventSourceARN": "arn:aws:kinesis:us-east-1:000000000000:stream/MyStream",
            "eventSource": "aws:kinesis",
            "eventVersion": "1.0",
            "eventName": "aws:kinesis:record",
            "invokeIdentityArn": "arn:aws:iam::000000000000:role/lambda-role",
            "awsRegion": "us-east-1",
            "kinesis":
            {
                "sequenceNumber": "49636781880378287419242807609792252425332679328602783746",
                "approximateArrivalTimestamp": 1672938988855.0,
                "data": "ZGF0YQ==",
                "partitionKey": "0",
                "encryptionType": "NONE"
            }
        }
    ]
}
```
